src/services/videomania/index.jsx

```
@@ -10,34 +10,58 @@ import Form from "../../components/form/Form";
-import { getVideoSubCategoryByCategory } from "../../app/features/videomania";
+import {
+  addVideoMania,
+  getVideoSubCategoryByCategory,
+} from "../../app/features/videomania";
 import { FaUpload } from "react-icons/fa";
 import videoIcon from "../../assets/videoIcon.svg";
 import { Toast } from "../../components/theme/Toast";
+import { uploadImage, uploadVideo } from "../../utils/common/cloudinary";
 
 export const AddVideoMania = ({
   setAddModal,
```

ESLint

[error] 'setAddModal' is missing in props validation

(react/prop-types)